source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+
+Basic
+-----
+
+.. image:: images/driver-station-labview-dashboard/basic.png
+
+The Basic tab contains a variety of pre-populated bi-directional controls/indicators which can be used to control the robot or display information from the robot. The SmartDashboard key names associated with each item are labeled next to the indicator with the exception of the Strings which follow the same naming pattern and increment from DB/String 0 to DB/String 4 on the left and DB/String 5 to DB/String 9 on the right. The LabVIEW framework contains an example of reading from the Buttons and Sliders in Teleop. It also contains an example of customizing the labels in Begin. For more detail on using this tab with C++\Java code see :doc:`using-the-labview-dashboard-with-c++-java-code`.
+
+Custom
+------
+
+.. image:: images/driver-station-labview-dashboard/custom.png
+
+The Custom tab allows you to add additional controls/indicators to the dashboard using LabVIEW without removing any existing functionality. To customize this tab you will need to create a Dashboard project in LabVIEW.
+
+Test
+----
+
+.. image:: images/driver-station-labview-dashboard/test.png
+
+The Test tab is for use with Test mode for teams using LabVIEW (Java and C++ teams should use the Java SmartDashboard when using Test Mode). For many items in the libraries, Input/Output info will be populated here automatically. All items which have ** next to them are outputs that can be controlled by the dashboard. To control an output, click on it to select it, drag the slider to set the value then press and hold the green button to enable the output. As soon as the green button is released, the output will be disabled. This tab can also be used to run and monitor tests on the robot. An example test is provided in the LabVIEW framework. Selecting this test from the dropdown box will show the status of the test in place of the slider and enable controls.
+
+Commands
+--------
+
+.. image:: images/driver-station-labview-dashboard/commands.png
+
+The Commands tab can be used with the Robot in Test mode to see which commands are running and to manually run commands for test purposes.
+
+Checklist
+---------
+
+.. image:: images/driver-station-labview-dashboard/checklist.png
+
+The Checklist tab can be used by teams to create a list of tasks to perform before or between matches. Instructions for using the Checklist tab are pre-populated in the default checklist file.
+
+Variables
+---------
+
+.. image:: images/driver-station-labview-dashboard/variables.png
+
+The Variables tab of the left pane shows all Network Tables variables in a tree display. The Variable Name (Key), Value and data type are shown for each variable. Information about the Network Tables bandwidth usage is also displayed in this tab. Entries will be shown with black diamonds if they are not currently synced with the robot.
+
+Record/Playback
+---------------
+
+.. image:: images/driver-station-labview-dashboard/record-playback.png
```

Comment:
Update picture for 2020

Reply:
Updated to: 
![image](https://user-images.githubusercontent.com/15172054/81518544-4032ca00-92f3-11ea-86cc-e05bc5b7402a.png)